source/includes/_datasets.md

```diff
@@ -1,21 +1,34 @@
 # Datasets
 
+Datasets are at the core of the platform. A dataset is composed of:
+
+- metadata like a title, a description and keywords, that make users able to discover it in the portal's catalog
+- configurations for processors, visualisation and security, which define the way data will be processed by the platform and made visible to users
+- and of course, the actual data you want to put on your portal (not available via the management API)
+
+Through the management API, you can:
```

Comment:
You should avoid using "you".
You could write: "Through the management API, it is possible to:"